src/geometry/types.rs

```diff
@@ -230,4 +314,26 @@ mod tests {
         // given, when, then
         assert_eq!(Vector::zero().normalized(), None)
     }
+
+    #[test]
+    fn rect_quadrant() {
+        // given
+        let mut rect = Rect { origin: Point { x: 0.0, y: 0.0 }, size: Size { width: 6.0, height: 8.0 } };
+
```

Comment:
It's good practice to write this over several lines. When you write it inline like thus it is hard to read.